packages/manager/modules/emailpro/src/emailpro.html

```diff
@@ -113,8 +113,10 @@ <h1 class="d-inline-block mb-0" data-ng-bind="exchange.displayName"></h1>
                         data-ng-if="messageDetails.length > 0"></strong>
             </div>
         </div>
-        <div class="tab-pane active platform"
-             data-ng-include="selectedTab && 'emailpro/' + kebabCase(selectedTab) + '/emailpro-' + kebabCase(selectedTab) + '.html'"></div>
+        <div ui-view>
```

Comment:
To keep consistency:
```suggestion
        <div data-ui-view>
```